site/en/blog/fedcm-origin-trial/index.md

```diff
@@ -6,15 +6,47 @@ authors:
 description: >
   A Web Platform API that allows users to login to websites with their federated accounts in a privacy preserving manner.
 date: 2022-04-25
-updated: 2022-06-20
+updated: 2022-07-29
 tags:
   - privacy
   - security
   - origin-trials
+  - identity
 ---
 
 {% Aside %}
 
+**Update, July 2022**
+
+Shipping FedCM is postponed to Chrome 106.
```

Comment:
IIRC we have never communicated publicly that we'd ship FedCM in 105, right? So it's not "postpone" :) More importantly, we have not decided when to ship it. Will probably get an answer in 2-3 weeks.

Reply:
Indeed. Removed this paragraph.